register/views.py

```diff
@@ -13,29 +15,43 @@
 from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
 from django.contrib.sites.shortcuts import get_current_site
 
-from .utils import token_generator
+from .utils import token_generator, forgot_password_token
 
 def home_page(request):
     return render(request, 'register/home.html')
 
+
+
 def register(request):
     if request.method == 'POST':
-        user_form = forms.UserForm(data=request.POST)
-
+        # user_form = forms.UserForm(data=request.POST)
+        user_form = forms.UserForm(request.POST)
+        
         if user_form.is_valid():
             user = user_form.save()
-            user.set_password(user.password)
+            password1 = user_form.cleaned_data.get('password1')
+            # password2 = user_form.cleaned_data.get('password2')
+            # print(password1)
+            user.last_name=request.POST['lname']
+            user.first_name=request.POST['fname']
+            print("##########################################################################")
+            print(password1)
+            print(user.last_name)
+            print(user.first_name)
+            user.set_password(password1)
             user.is_active = False
             user.save()
-
             # encoding user id to send over the network
 
             uidb64 = urlsafe_base64_encode(force_bytes(user.pk))
-            domain = get_current_site(request).domain
+            # Use the code below to get domain name
+            # domain = get_current_site(request).domain
+            # use the code below while running on local server since the above code is not changing is_active to True
+            domain='127.0.0.1:8000'
```

Comment:
this is the hardcoded domain. Use `get_current_site(request).domain` to generate domain dynamically

Reply:
Can I comment out the code on line 50? or should I remove it?